_posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md

```diff
@@ -0,0 +1,126 @@
+| 14 | 222.54 - 228.52 | HolidayCheck Agent | Genau, das gilt für die erste Reise. Sie können es heute abschließen, aber erst in zwei Wochen eine Reise buchen. Das ist ganz hinüberlassen.                                                                                                                                                                                                                                                                                                                                                                                   | 0.997, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 0.996, 1.000, 0.999, 1.000, 0.999, 0.998, 0.815, 0.999, 0.958, 1.000, 0.999, 1.000, 0.999, 0.961, 0.808, 0.881, 0.629                                                                                                                                                                                                                                                                                                                                                                                                  |
+| 15 | 229.34 - 237.72 | Customer           | Wunderbar. Prima. Ach, da haben Sie mir aber schon sehr weitergeholfen. Wunderbar. Herzlichen Dank. Sehr gerne. Und auf Wiederhören. Bis dann. Tschüss. Tschüss.                                                                                                                                                                                                                                                                                                                                                                | 0.974, 1.000, 0.947, 1.000, 1.000, 0.999, 1.000, 1.000, 1.000, 1.000, 0.925, 1.000, 0.999, 1.000, 0.993, 0.999, 0.907, 0.986, 0.998, 0.859, 0.993, 0.972, 0.943                                                                                                                                                                                                                                                                                                                                                                                                         |
+
+**Step 2:** Even though we use self-hosted LLMs, but since raw transcripts can contain some form of personal information, we always do multiple scans locally using [Presidio](https://github.com/microsoft/presidio) and some custom prompts with Mistral. This really helps us remove majority of the personal information without loosing bits of conversation.
```

Comment:
remove the majority of the personal information without losing too many bits of the conversion